examples/integrations/itk/SelectROI.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # Request the highest level explicitly in case that is not what is loaded.
</pre><p>To make the connection clearer, let's use <code>loaded_scale</code> here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/itkwidgets/pull/703/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Sounds good!